pkl-core/src/main/java/org/pkl/core/util/yaml/YamlEmitter.java

```diff
@@ -210,6 +212,11 @@ public final void emit(boolean value) {
     builder.append(value);
   }
 
+  public final void emit(byte[] value) {
+    builder.append("!!binary ");
+    builder.append(Base64.getEncoder().encodeToString(value));
```

Comment:
Hmm.. should we encode this in RFC 2045 style? E.g. wrap at 76 chars with CRLF line breaks?

Seems like this is the typical way that this gets encoded in YAML: https://yaml.org/type/binary.html